template.tpl

```diff
@@ -22,7 +22,7 @@ ___INFO___
   },
-  "description": "The tag utilizes the Amplitude Browser SDK 2.0 for data collection. Check the Breaking Changes List \u003ca herf\u003d\"https://www.docs.developers.amplitude.com/data/sources/google-tag-manager-client\"\u003ehere\u003c/a\u003e.",
+  "description": "Amplitude is a powerful product analytics platform that enables you to build better products by tracking and understanding user behavior. This template facilitates the deployment of the Amplitude Browser 2.0 SDK and the utilization of its API on your site. For more information, visit our GitHub repository at https://github.com/amplitude/amplitude-browser-sdk-gtm-template.",
```

Comment:
1. Link is not working correctly in the GTM gallery, so use a direct link here.
2. The github repository will only show when you first install the template from the gallery, add provide the github link here for those who want to get more info while updating to the latest version. 
![Screenshot 2024-05-17 at 5 37 23 PM](https://github.com/amplitude/amplitude-browser-sdk-gtm-template/assets/13028329/53d88a3e-4565-4cb5-95c4-8e65bf9d4888)

![Screenshot 2024-05-17 at 5 43 07 PM](https://github.com/amplitude/amplitude-browser-sdk-gtm-template/assets/13028329/e9a6edf4-1f96-46fd-8c77-c6a792c37b80)